docs/developing.rst

```diff
@@ -0,0 +1,58 @@
+Developing
+==========
+
+Install the requirements
+
+::
+
+    $ pip install -r requirements.txt
+
+(note that lxml, used in the tests, requires that libxslt and libxml2 be
```

Comment:
Nevermind about Python development package. I think if you are in the development environment to run tests, you should be okay.